lib/index.js

```diff
@@ -218,7 +218,7 @@ internals.Yar = class {
 
         let messages;
         this._isModified = true;
-        this._store._flash = this._store._flash || {};
+        this._store._flash = this._store._flash ?? {};
```

Comment:
You could do that as well
```suggestion
        this._store._flash ??= {};
```